Earthfile

```diff
@@ -350,7 +350,7 @@ base-image:
     END
 
 
-    IF [[ "$FLAVOR" =~ ^alpine.* ]] || [[ "$FLAVOR" =~ .*-arm-rpi$ ]]
+    IF [[ "$FLAVOR" =~ ^alpine.* ]]
```

Comment:
I wonder how we could document such kind of information ... like it would be nice if instead of checking here on the flavor, there could be some sort of file where we check "hasDracut?" and if the current flavor happens to be listed there it's true

Reply:
not only that but all the differences between distros on if they auto-lin the Image or we have to do it, if its a symlink or a regular file. pita!

Reply:
in any case, alpine wont have dracut unless you install it for testing so maybe we can end up with ALL of them just running dracut somehow :D